### ISSUE TITLE
List recent scorigamis

### ISSUE THREAD

**Issue by Slycir** [ACMerriman/scorigami#37]
Not an issue but a suggestion: list the most recent scorigamis below this weeks matches.

**Comment by fynsta** [comment]
Yeah, it would be nice to see how many scorigamis there have been this year

**Comment by SimonHamermesh** [comment]
last night 01/08/22 - dallas 51 philadelphia 26

**Comment by ACMerriman** [comment]
If you select "First Game" in the dropdown, you can see all of the scorigamis that happened each year
